### PR TITLE
Incorrectly rendered table

### PR DESCRIPTION
**Pull request by chunduriv**
The API docs for multiple pages in keras.applications are not rendered properly, see the tables on:

https://www.tensorflow.org/api_docs/python/tf/keras/applications/nasnet
https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet_v2
https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet

Fixes https://github.com/keras-team/keras/issues/16727

### CODE REVIEW

File: keras/applications/nasnet.py [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
 
 The below table describes the performance on ImageNet 2012:
---------------------------------------------------------------------------------
-      Architecture       | Top-1 Acc | Top-5 Acc |  Multiply-Adds |  Params (M)
---------------------------------------------------------------------------------
-|   NASNet-A (4 @ 1056)  |   74.0 %  |   91.6 %  |       564 M    |     5.3    |
-|   NASNet-A (6 @ 4032)  |   82.7 %  |   96.2 %  |      23.8 B    |    88.9    |
---------------------------------------------------------------------------------
+---------------------------------------------------------------------------
+
```
Comment (fchollet) [inline keras/applications/nasnet.py:30]: Please remove blank line

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Comment by mihirparadkar** [comment]
Please fix the lint errors.

**Review by gbaned — APPROVED** [review]

**Comment by udaylunawat** [comment]
This issue still persists!

**Pull request merged**